extensions/control-plane/api/management-api/policy-definition-api/src/main/java/org/eclipse/edc/connector/api/management/policy/PolicyDefinitionApi.java

```diff
@@ -90,4 +91,14 @@ public interface PolicyDefinitionApi {
     )
     void deletePolicy(String id);
 
+    @Operation(description = "Updates an existing Policy, If the Policy is not found, no further action is taken",
```

Comment:
Please see the other comment about the behavior

Reply:
Sure, I have changed the description